llvm/test/SYCLPropagateAspectUsage/aggregate_types.ll

```diff
@@ -0,0 +1,71 @@
+; RUN: opt --PropagateAspectUsage < %s -S
+; | FileCheck %s
+;
+; Test checks that no metadata propagates because MyStruct
+; isn't used inside functions.
+
+; Optional
+%A = type { i32 }
```

Comment:
`clang` emits struct types with `struct.` or `class.` prefix - this helps to distinguish types from regular variables, I suggest to re-use this approach in tests as well

Reply:
clang does it. But it doesn't influence pass behavior and the current version is simpler. Also I can see similar approach in other tests. [Example](https://github.com/llvm/llvm-project/blob/main/llvm/test/Analysis/BasicAA/128-bit-ptr.ll)